portfolio/src/main/webapp/index.html

```diff
@@ -105,7 +105,7 @@ <h4 class="card-header">
       </div>
     </div>
     <div class="mb-3">
-      <button onclick="addRandomQuote()" class="btn btn-secondary">Random quote</button>
+      <button onclick="addRandomQuote()" class="btn btn-secondary mr-3">Random quote</button>
```

Comment:
what is this part?

Reply:
This adds a margin between the quote button and the quote itself. 'mr-3' means 'margin right 1.0 * specific value depending on the size of the screen'